app/js/components/drugOrderEntry/addForm/AddForm.jsx

```diff
@@ -33,7 +33,7 @@ export class AddForm extends React.Component {
     },
     draftOrder: {},
     orders: [],
-    orderNumber: '0',
+    stateOrderNumber: '0',
```

Comment:
What is the reason behind renaming this to stateOrderNumber?

Reply:
orderNumber exist in both props and state, so to avoid that I had to rename it to stateOrderNumber as suggested by @alexmochu.

Reply:
What does stateOrderNumber mean?

Reply:
its the index position of the order in the draft orders array. So in this context i called it stateOrderNumber to differentiate between the orderNumber in the state and props. As it will be a problem here
    `        if (this.state.draftOrder.action === 'NEW') {
                   setOrder('DRAFT', stateOrderNumber);
              } else {
                   setOrder('DRAFT_EDIT', orderNumber);
              }
     `